analyses/lancet_padding_window/results/all_samples_with_var_comparison

```diff
@@ -0,0 +1,20 @@
+300-600	WHCGBM_0000000011	 1383
+200-400	WHCGBM_0000000011	 1351	 4 3_prime_UTR_variant 13 downstream_gene_variant 34 intergenic_region 32 intron_variant 1 missense_variant 2 non_coding_transcript_exon_variant 1 synonymous_variant 9 upstream_gene_variant
```

Comment:
This is quite helpful! Perhaps you can make a bar graph summarizing the missing variants by category. Basically, per sample, per window? If possible maybe choose most deleterious hit for each call, rather than listing all possible.

Reply:
After creating consensus comparison, it's small enough not to need to be as fine-grained. We can ignore this request